src/app/lib/logger.const.js

```diff
@@ -21,6 +21,7 @@ const logCodes = {
   MEDIA_MISSING_FIELD: 'media_missing_field',
   MEDIA_PLAYER_STATUS: 'media_player_status',
   RADIO_EPISODE_EXPIRED: 'radio_episode_expired',
+  RADIO_FIELD_MISSING: 'radio_field_missing',
```

Comment:
Could this be more consistent with `MEDIA_MISSING_FIELD`?
```suggestion
  RADIO_MISSING_FIELD: 'radio_missing_field',
```

Reply:
👍